From 64b0d66684f7c0f63681f9bc844f83402efc386b Mon Sep 17 00:00:00 2001
From: Markos Chandras <markos.chandras@imgtec.com>
Date: Wed, 24 Jul 2013 13:21:12 +0000
Subject: [PATCH 100/105] MIPS: EVA: Don't write the EVA bit in the config5 CP0 register

The EVA bit (bit 28) in config5 CP0 is read-only so don't try
to write it.

Signed-off-by: Markos Chandras <markos.chandras@imgtec.com>
---
 arch/mips/mti-malta/malta-setup.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/mips/mti-malta/malta-setup.c b/arch/mips/mti-malta/malta-setup.c
index cff9a74..db1178b 100644
--- a/arch/mips/mti-malta/malta-setup.c
+++ b/arch/mips/mti-malta/malta-setup.c
@@ -352,7 +352,7 @@ void __init plat_eva_setup(void)
 	back_to_back_c0_hazard();
 
 	val = read_c0_config5();
-	write_c0_config5(val|MIPS_CONF5_K|MIPS_CONF5_CV|MIPS_CONF5_EVA);
+	write_c0_config5(val|MIPS_CONF5_K|MIPS_CONF5_CV);
 	back_to_back_c0_hazard();
 
 	printk("Enhanced Virtual Addressing (EVA) active\n");
-- 
1.7.1

